packages/donor/src/screens/contact/ContactScreen.tsx

```diff
@@ -0,0 +1,141 @@
+import Text from "../../components/basic/Text";
+import HeaderSection from "../../components/HeaderSection";
+import styles from "./styles.module.scss";
+import ZMScreen from "../../components/basic/ZMScreen/ZMScreen";
+
+import PhoneIcon from "../../assets/icons/phone.svg";
+import MailIcon from "../../assets/icons/mail.svg";
+
+import React, { ReactNode } from "react";
+
+export default function ContactScreen() {
```

Comment:
Do you want to add a story to this screen?

Shouldn't be too complicated

Reply:
Done.